Dockerfile

```diff
@@ -29,8 +28,6 @@ ARG ZOOM_URL=https://zoom.us/client/latest/zoom_amd64.deb
 RUN curl -sSL $ZOOM_URL -o /tmp/zoom_setup.deb
 RUN dpkg -i /tmp/zoom_setup.deb
 RUN apt-get -f install
-RUN rm /tmp/zoom_setup.deb \
-  && rm -rf /var/lib/apt/lists/*
```

Comment:
This won't necessarily be pointless in the future. There is the experimental `--squash` option that would allow this line to actually decrease the image size.

Reply:
True. Would save 40MB + 112MB eventually. You could save that now by squashing all the RUNs together but would sacrifice readability - not a big savings on 1.21GB image anyway.

Nice work - thanks for putting this together.

Reply:
Oh sorry, I'm not the maintainer. I've just been very interested in this repo lately, figured I had two cents to give :sweat_smile:

Reply:
Both PR and help for reviewing are much appreciated :)